cyber-security/modules/gds_security_audit_role/inline_policies.tf

```diff
@@ -0,0 +1,27 @@
+data "aws_iam_policy_document" "support_inline_policy_document" {
+  statement {
+    effect    = "Allow"
+    actions   = ["support:*"]
```

Comment:
Shouldn't this be `support:Describe*`?
https://docs.aws.amazon.com/IAM/latest/UserGuide/list_awssupport.html#awssupport-resources-for-iam-policies

Reply:
Unfortunately it can't be. We tried a while back when we built the csw-client-role policy. Weirdly you get an access denied for DescribeTrustedAdvisorCheckResult with anything other than support:* - I guess it may have been fixed since - I admit I've not checked again recently. Will do first thing tomorrow.

Reply:
I've tested this morning with support:Describe* and it does now work. I'm pretty sure it didn't work however long ago it was we setup the client role but I guess things at AWS change. I'm pushing a commit to make the change as requested.

Reply:
I'm pretty sure I remember it not working in the past as well - nice to see AWS actually fixing something :)

Reply:
https://docs.aws.amazon.com/IAM/latest/UserGuide/list_awssupport.html